README.md

```diff
@@ -49,9 +47,16 @@ Coming soon, will be live by December 6. You can use manually in the mean time p
 You can manually install the library by cloning this repo and creating a Podfile that references your local clone of it like below:
 
 ``` ruby
-  pod 'Amplify', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'AWSPluginsCore', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'CoreMLPredictionsPlugin', :path => '~/Projects/Amplify/amplify-ios'
-  pod 'AmplifyPlugins/AWSPredictionsPlugin', :path => '~/Projects/Amplify/amplify-ios'
+  pod 'Amplify', :path => '~/amplify-ios'
+  pod 'AWSPluginsCore', :path => '~/amplify-ios'
+  pod 'CoreMLPredictionsPlugin', :path => '~/amplify-ios'
+  pod 'AWSPredictionsPlugin', :path => '~/amplify-ios'
+  pod 'AmplifyPlugins/AWSAPIPlugin', :path => '~/amplify-ios'
```

Comment:
I suppose this is valid for people using Development pods; do we want to update the instructions above for users using Cocoapods, Carthage, etc, since the deadline is now a week past?

Reply:
i'll have to run through how to install via Carthage before updating the README. This seems like a good-first-issue opportunity

Reply:
https://github.com/aws-amplify/amplify-ios/issues/280

Reply:
I agree that we're probably not done with the Carthage instructions, but that shouldn't stop us from making the change on the other items right?